src/views/InvolvementsAll/index.jsx

```diff
@@ -173,86 +169,104 @@ const InvolvementsAll = () => {
-                  backgroundColor: gordonColors.primary.blue,
-                  color: gordonColors.neutral.grayShades[50],
-                }}
-              />
-              <CardContent>
-                {loading ? (
+            <TabContext value={tabIndex}>
+              <Grid item xs={16} marginTop={2}>
+                <TabList
+                  onChange={handleChangeTab}
+                  aria-label="involvements tabs"
+                  variant="fullWidth"
+                  centered
+                  indicatorColor="secondary"
+                >
+                  <Tab label="Personal" className={styles.tab} value="1" />
+                  <Tab label="Requests" className={styles.tab} value="2" />
+                  <Tab label="All" className={styles.tab} value="3" />
+                </TabList>
+              </Grid>
+              <TabPanel value="1">
+                {loadingProfile ? (
                   <GordonLoader />
                 ) : (
-                  <InvolvementsGrid
-                    involvements={myInvolvements}
-                    sessionCode={selectedSession}
-                    noInvolvementsText={myInvolvementsNoneText}
-                  />
+                  profile && (
+                    <Grid item xs={12} lg={12}>
+                      {loading ? (
+                        <GordonLoader />
+                      ) : (
+                        <InvolvementsGrid
+                          involvements={myInvolvements}
+                          sessionCode={selectedSession}
+                          noInvolvementsText={myInvolvementsNoneText}
+                        />
+                      )}
+                    </Grid>
+                  )
                 )}
-              </CardContent>
-            </Card>
-          </Grid>
-        )
-      )}
-
-      {/* All Involvements (public) */}
-      <Grid item xs={12} lg={8}>
-        <Card>
-          <CardHeader
-            title={`${involvementSessionText} Involvements`}
-            style={{
-              backgroundColor: gordonColors.primary.blue,
-              color: gordonColors.neutral.grayShades[50],
-            }}
-          />
-          <CardContent>
-            {loading ? (
-              <GordonLoader />
-            ) : (
-              <InvolvementsGrid
-                involvements={involvements}
-                sessionCode={selectedSession}
-                noInvolvementsText="There aren't any involvements for the selected session and type"
-              />
-            )}
+              </TabPanel>
+              <TabPanel value="2">
+                <Grid
+                  item
+                  xs={12}
+                  lg={12}
+                  xl={12}
+                  container
+                  alignItems="center"
+                  justifyContent="center"
+                >
+                  {!isOnline ? null : loadingProfile ? (
+                    <GordonLoader />
+                  ) : (
```

Comment:
I think this is already inside of a check for `loadingProfile`. Correct me if I'm misreading the indentations.